### PR TITLE
Update contributing guides

### LINKED ISSUE #670: Update Contributing Docs for Styles and Builds
## Summary

Updating the contributing docs to account for style changes, how to build the docs locally

### Basic example

Adding steps for:
- .vscode dictionary for aiding in US/UK spelling
- adding versioning information for docs dependencies
- how to format images for consistency
- Title Case styles for headings

### Motivation

Adding to help the team scale for consistency in edits and speed by which we can make changes

### PR DESCRIPTION
**Pull request by SteveLasker**
Resolves #670

Includes:
- spelling errors
- dictionary to help avoid spelling errors while editing with vscode
- setting the npm version locally for local testing

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by SteveLasker** [comment]
cc: @beekman33

File: content/contributing/getting-started/pulling-and-building-rkvst-docs/index.md [hunk 1]
```diff
@@ -15,39 +15,39 @@ toc: true
...
     ```  
-5. To run a local RKVST Docs Server it is advisable to use the `rkvst-doks` wrapper.
+1. To run a local RKVST Docs Server it is advisable to use the `rkvst-doks` wrapper
     ```bash
-    rkvst-doks start
+    ./rkvst-doks start
     ```
     This will build a local version of the server that can be accessed at [http://localhost:1313](http://localhost:1313).
     You can keep this running in the background while making any edits and the developer server should automatically pick up your changes for you.
     {{< note >}}
```
Comment (pcnorth) [inline content/contributing/getting-started/pulling-and-building-rkvst-docs/index.md:50]: Delete indentation to fix the display of the Note

File: content/contributing/adding-content/adding-articles-to-rkvst-docs/index.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ toc: true
     ```
     Here you can see we have specified the `contributing` area, the `adding content` section and the name of our new article with a non-underscored `index.md` to indicate this is an article
     {{< note >}}
```
Comment (pcnorth) [inline content/contributing/adding-content/adding-articles-to-rkvst-docs/index.md:24]: For some reason the indentation of this Note block breaks how it is displayed in my browser. Deleting some of the indentation fixes it.

File: content/contributing/adding-content/adding-content-sections-to-rkvst-docs/index.md [hunk 3]
```diff
@@ -22,10 +22,10 @@ toc: true
     ```
     Here you can see we have specified the `platform` area, the name of our new section and we use the underscored `_index.md` to indicate it is an indexing section of content
     {{< note >}}
-    **Note:** The name of each folder should be sluggified
```
Comment (pcnorth) [inline content/contributing/adding-content/adding-content-sections-to-rkvst-docs/index.md:25]: Again, the indentation breaks this Note

**Review by pcnorth — CHANGES_REQUESTED** [review]
The indentation of some of the Note sections is preventing them from displaying correctly in my browser.
I found that deleting 1 or 2 indentations for the whole block fixes it.

**Comment by SteveLasker** [comment]
Thanks, @pcnorth, I believe I've resolved all the formatting and additional limiting fixes.

**Comment by mmwilder26** [comment]
Hello @SteveLasker   . .  . I jumped on this branch and see {{< note >}} in clear text on some of the .md files.  Is it just me seeing this or. . .?  I also see {{ < warning >}}  in clear text in one of the .md files.  Yes, did see the comment from Phil  . . . maybe these have been corrected and it's just me. . . thought I make a note that the formatting text displays for me.

**Review by pcnorth — APPROVED** [review]

**Comment by pcnorth** [comment]
I pulled the latest changes into my local version, all the note, caution and warning sections are displaying ok.

**Pull request merged**